client/cmd/dexc-desktop/README.md

```diff
@@ -7,11 +7,29 @@ The dexc-desktop client is a cross-platform desktop application for the DEX clie
 ## Debian
 Build with `./pkg/pkg-debian.sh`. The deb archive is located in **./build**.
 
-For development, you'll need to install the WebKit Development Libraries.
-`apt install libgtk-3-dev libwebkit2gtk-4.0-dev`
+For development, you'll need to install the WebKit Development Libraries and the
+standard build toolchain:
+
+```bash
+apt install libgtk-3-dev libwebkit2gtk-4.0-dev build-essential
+```
```

Comment:
I tried this on a fresh installation and I think you need to build the site first at `client/webserver/site` as well? Not sure if worth mentioning here.

Reply:
Yes, good point - in development `site/` needs to be built indeed, on the release branch it's not necessary.